.travis.yml

```diff
@@ -3,8 +3,8 @@ sudo: false
 
 go_import_path: go.uber.org/fx
 go:
-  - 1.8
-  - 1.9
+  - "1.9"
+  - "1.10"
```

Comment:
strings necessary all of the sudden?

Reply:
Yeah, see PR description. Just `1.10` is interpreted as a float in YAML, so it becomes `1.1`.